catalog/spatial/registry/registry-admin-modules/registry-admin-poller-service/src/main/java/org/codice/ddf/registry/admin/poller/AdminPollerServiceBean.java

```diff
@@ -200,9 +211,40 @@ public boolean sourceStatus(String servicePID) {
     }
 
     @Override
-    public List<Map<String, Object>> allSourceInfo() {
+    public List<List<Map<String, Object>>> allSourceInfo() {
+        List<List<Map<String, Object>>> allSourceInfo = new ArrayList<>();
+        List<Map<String, Object>> sourceInfo = allMetatypeInfo(
+                SOURCE_FILTER);
+
+        allSourceInfo.add(sourceInfo);
+
+        List<Map<String, Object>> registryMetacardInfo = new ArrayList<>();
+
+        try {
+            List<RegistryPackageType> registryMetacardObjects =
+                    federationAdminService.getRegistryObjects();
+            for (RegistryPackageType metacardObject : registryMetacardObjects) {
+                registryMetacardInfo.add(RegistryPackageWebConverter.getRegistryObjectWebMap(
+                        metacardObject));
+            }
+        } catch (FederationAdminException e) {
+            LOGGER.warn("Couldn't get remote registry metacards '{}'", e);
+        }
+
+        allSourceInfo.add(registryMetacardInfo);
+
+        return allSourceInfo;
+    }
+
+    @Override
+    public List<Map<String, Object>> allRegistryInfo() {
```

Comment:
Just curious what the difference is between allRegistryInfo() and allSourceInfo()...it looks like allSourceInfo() is all the sources and registry info. What is the value in the other method, and do we need one for just sources (not registry) ?